PyInstaller/building/makespec.py

```diff
@@ -216,7 +216,9 @@ def __add_options(parser):
     g = parser.add_argument_group('Windows and Mac OS X specific options')
     g.add_argument("-c", "--console", "--nowindowed", dest="console",
                    action="store_true", default=True,
-                   help="Open a console window for standard i/o (default)")
+                   help="Open a console window for standard i/o (default). "
+                        "Caution: this option will be overwritten to use "
+                        "the --noconsole option if using a '*.pyw' file.")
```

Comment:
What about "Caution: This option will have no effect if freezing a '.pyw' file."?
And for --windowed something like "This option will be set automatically if freezing a '.pyw' file."?

Reply:
That sounds better to me!